src/Razor/src/Microsoft.NET.Sdk.Razor/ReferenceResolver.cs

```diff
@@ -62,7 +71,7 @@ private DependencyClassification Resolve(ClassifiedAssemblyItem classifiedItem)
                     DependencyClassification.MvcReference :
                     DependencyClassification.DoesNotReferenceMvc;
 
-                return classifiedItem.DependencyClassification;
+                return;
```

Comment:
We did this weird thing where the value was being assigned to the item and then we returned it. I introduced a bug when I was messing with this by returning a value, but not assigning it to a field. Removing these makes it a bit clearer.